packages/SystemsThinkingWorkbench-Core.package/STWPlayfield.class/instance/addSimulationButtons.st

```diff
@@ -0,0 +1,15 @@
+initialization
+addSimulationButtons
+	
+	self playButton: STWButton new.
+	self playButton contents: 'Start'.
+	self playButton clickHandler: [ self toggleSimulation. ].
```

Comment:
Well, it would be a two element cascade, as i need to create and set the contents of the morph, then i need to store it as play button, then i need to add it as submorph, to then set its position.